walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -230,11 +228,10 @@ public void ignoresPeopleNotAttending() {
     Collection<Event> events = Arrays.asList(new Event("Event 1",
         TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES), Arrays.asList(PERSON_A)));
     MeetingRequest request = new MeetingRequest(Arrays.asList(PERSON_B), DURATION_30_MINUTES);
-
     Collection<TimeRange> actual = query.query(events, request);
     Collection<TimeRange> expected = Arrays.asList(TimeRange.WHOLE_DAY);
-
     Assert.assertEquals(expected, actual);
+    
```

Comment:
Maybe there's an issue with the diff, but it doesn't look like you've added any tests (you mentioned just one not being there, but it looks like this is just the default you were provided).

Reply:
Yeah that was my bad. Done!